tools/go.mod

```diff
@@ -1,8 +1,8 @@
 module github.com/submariner-io/shipyard/tools
 
-go 1.22.0
+go 1.23.0
 
-toolchain go1.22.2
+toolchain go1.23.8
 
```

Comment:
```suggestion
```

to avoid unnecessary downloads during the build (the toolchain in our Shipyard base image is 1.23.7).

(The `toolchain` directive records the version used to make a change; it shouldn’t reflect an actual hard requirement. It’s an attempt to make `go.mod` useful for reproducibility…)